bootstrap-gruntwork-installer.sh

```diff
@@ -245,6 +254,10 @@ function bootstrap {
         no_sudo="$2"
         shift
         ;;
+      --enable-retries)
+        retry_number=10
```

Comment:
Did you  mean to use `ENABLED_CURL_RETRY_NUMBER` here?

Reply:
I certainly did. Fixed.